packages/badge/Badge.test.tsx

```diff
@@ -7,12 +7,52 @@ describe( Badge.is, () => {
 
   it( 'should render', () => {
 
-    console.warn('Missing tests for Badge!');
-
     return mount(
       <Badge/>
-    ).wait();
+    ).wait((element) => element.has('c-badge') );
 
   });
 
+  it( 'should render rounded', () => {
+
+    return mount(
+      <Badge rounded>Rounded</Badge>
+    ).wait((element) => element.has('c-badge--rounded'));
+
+  } );
+
+  it( 'should render ghosted', () => {
+
+    return mount(
+      <Badge ghost>Ghosted</Badge>
+    ).wait((element) => element.has('c-badge--ghost'));
+
+  } );
+
+  it( 'should render with blaze colors', () => {
+
+    return mount(
+      <div>
+        <Badge color="brand">Brand</Badge>
+        <Badge color="info">Info</Badge>
+        <Badge color="warning">Warning</Badge>
+        <Badge color="success">Success</Badge>
+        <Badge color="error">Error</Badge>
+      </div>
+    ).waitFor((wrapper) => {
+      const badges = wrapper.all(Badge);
+      return (
+        !!badges[0].node.shadowRoot
```

Comment:
I'm not sure if we need to test all colors a we have helper for that, also testing just first shadowRoot doesn't feels right. I would either return true or check all.